tests/forte/data/data_store_test.py

```diff
@@ -126,19 +128,34 @@ def setUp(self) -> None:
         self.reference_type_attributes = {
```

Comment:
Looks like `data_store_test.py` and `data_store_serialization_test.py` share a lot of resources. Maybe we should just keep single source of truth. Note that they do have difference and some of the difference may be intentional.

Reply:
Should we keep that a part of this PR?

Reply:
I think it can be another PR and it's not a prority right now.